src/components/table/JadeTable.jsx

```diff
@@ -115,9 +120,26 @@ export class JadeTable extends React.PureComponent {
     });
   };
 
-  handleArrayValues = (value) => {
+  handleArrayValues = (value, column) => {
     if (_.isArray(value)) {
-      return `[${value.map((obj) => obj.v).join(', ')}]`;
+      const returnValue = [];
+      if (column.mode === COLUMN_MODES.REPEATED) {
+        returnValue.push(<span key="start">[</span>);
+      }
+      returnValue.push(
+        ...value.flatMap((v, i) => [
+          v,
+          i < value.length - 1 ? (
+            <span key={`sep-${i}`}>
+              ,<br />
+            </span>
+          ) : undefined,
+        ]),
+      );
```

Comment:
not sure if i completely understand what this is doing, is this just adding commas until the end of the array?

Reply:
Yeah...adding commas to everything except the last element